web/src/components/pages/dashboard/dashboard.tsx

```diff
@@ -63,8 +66,20 @@ const TopBar = ({ dashboardLocale }: { dashboardLocale: string }) => {
   }, []);
 
   return (
-    <div className="top-bar">
+    <div className={`top-bar ${isChallengeTabVisible ? 'challenge' : ''}`}>
       <nav>
+        {/* only visible for users enrolled in challenge*/}
+        {isChallengeTabVisible ? (
```

Comment:
Nitpick: if you're conditionally rendering a single component, I think the preferred style is `{isChallengeTabVisible && ( ...` rather than a ternary. See https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator.